src/Shouldly.Shared/ShouldlyExtensionMethods/ShouldBeEnumerableTestExtensions.cs

```diff
@@ -47,6 +47,15 @@ public static void ShouldContain<T>(this IEnumerable<T> actual, [InstantHandle]
         {
             ShouldContain(actual, elementPredicate, () => null);
         }
+        public static void ShouldContainMatchingCount<T>(this IEnumerable<T> actual, [InstantHandle] Expression<Func<T, bool>> elementPredicate, int expectedCount)
```

Comment:
Whoops, didn't see this comment. I'll rename; is this also causing a test failure?